api/package.json

```diff
@@ -72,6 +72,7 @@
     "factory-girl": "^5.0.4",
     "jsdom": "^10.1.0",
     "nodemon": "1.18.9",
-    "sinon": "^9.0.2"
+    "sinon": "^9.0.2",
+    "aws-sdk-mock": "^5.4.0"
```

Comment:
AWS provides a mock library for testing it's services as opposed to manually writing the mocks ourselves.  Used for testing the SES driver